packages/tinybird/src/client.ts

```diff
@@ -1,41 +1,28 @@
 import type { Tinybird } from "@chronark/zod-bird";
-import * as z from "zod";
+
+import {
+  tbBuildResponseList,
+  tbIngestPingResponse,
+  tbParameterResponseList,
+} from "./validation";
 
 // REMINDER:
 // const tb = new Tinybird({ token: process.env.TINYBIRD_TOKEN! });
 
-export const tinyBirdEventType = z.object({
-  id: z.string(),
-  workspaceId: z.string(),
-  pageId: z.string(),
-  monitorId: z.string(),
-  timestamp: z.number().int(),
-  statusCode: z.number().int(),
-  latency: z.number().int(), // in ms
-  url: z.string(),
-  metadata: z.string().optional().default(""),
-  region: z.string().min(4).max(4), // TODO: object + stringify json
-});
-
-// TODO: think of a better name `publishHttpResponse`
 export function publishPingResponse(tb: Tinybird) {
   return tb.buildIngestEndpoint({
     datasource: "ping_response__v2",
-    event: tinyBirdEventType,
+    // @ts-ignore TODO: this is caused by the metadata even though the
+    // error says something regarding _input.cronTimestamp
+    event: tbIngestPingResponse,
```

Comment:
Having some really unexpected and weird problem. Need to investigate further. `pnpm tsc` doesn't through any error though...